data_steward/tools/combine_ehr_rdr.py

```diff
@@ -361,10 +381,58 @@ def load_mapped_person():
     query(q, 'person', write_disposition='WRITE_TRUNCATE')
 
 
+def copy_deid_map_table(pid_rid_mapping_table, project_id, dataset_id, client):
+    job_config = bq.bigquery.QueryJobConfig(destination=pid_rid_mapping_table)
+    q = COPY_PID_RID_QUERY.render(
+        project=project_id,
+        dataset=dataset_id,
+        pid_rid_table=combine_consts.PID_RID_MAPPING_TABLE)
+
+    query_job = client.query(q, job_config=job_config)
+    query_job.result()
+
+
+def load_deid_map_table():
+
+    # Create _pid_rid_mapping table
+    project_id = app_identity.get_application_id()
+    client = bq.get_client(project_id)
+    deid_map_mapping_table = f'{project_id}.{bq_utils.get_combined_dataset_id()}.{combine_consts.DEID_MAP_TABLE}'
+    bq.create_tables(client,
+                     project_id, [deid_map_mapping_table],
+                     exists_ok=False)
+
+    # copy data to _pid_rid_mapping_table
+    if bq_utils.table_exists(combine_consts.DEID_MAP_TABLE,
+                             dataset_id=bq_utils.get_rdr_dataset_id()):
+        copy_deid_map_table(deid_map_mapping_table, project_id,
+                            bq_utils.get_rdr_dataset_id(), client)
+        logging.info(
+            f"{project_id}.{bq_utils.get_rdr_dataset_id()}.{combine_consts.DEID_MAP_TABLE} \
+            copied to the table {deid_map_mapping_table}")
+
+    # This else condition is a stop gap fix until RDR starts sending the _pid_rid_mapping table
+    elif bq_utils.table_exists(
+            combine_consts.PID_RID_MAPPING_TABLE,
+            dataset_id=combine_consts.PIPELINE_TABLES_DATASET):
+        copy_deid_map_table(deid_map_mapping_table, project_id,
+                            combine_consts.PIPELINE_TABLES_DATASET, client)
+        logging.info(
+            f"{project_id}.{combine_consts.PIPELINE_TABLES_DATASET}.{combine_consts.PID_RID_MAPPING_TABLE} \
+                    copied to the table {deid_map_mapping_table}")
```

Comment:
I'm not sure what advantage this stop gap solution is creating.  We have already copied the `_deid_map` table to a `pipeline_tables.pid_rid_mapping`.  The deid code will not be updating that table (some other code will be developed for that), but deid can use it as if it exists because it has already been moved.  Is there a design decision I have overlooked?